mule-extensions-xml-archetype/src/test/resources/projects/basic-with-spaces/reference/pom.xml

```diff
@@ -132,7 +133,19 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-javadoc-plugin</artifactId>
-                    <version>${maven.javadoc.plugin.version}</version>
+                    <version>${maven.javadoc.plugin}</version>
+                    <executions>
+                        <execution>
+                            <id>attach-javadocs</id>
+                            <goals>
+                                <goal>jar</goal>
+                            </goals>
+                        </execution>
+                    </executions>
+                    <configuration>
+                        <doclint>none</doclint>
+                        <additionalparam>${xDocLint}</additionalparam>
```

Comment:
Do we need both `doclint` and `additionalparam` parameters?

Reply:
We need only `doclint`